src/pip/_internal/locations.py

```diff
@@ -75,16 +75,16 @@ def get_src_prefix():
 except AttributeError:
     user_site = site.USER_SITE
 
+bin_user = sysconfig.get_path('scripts')
+
 if WINDOWS:
     bin_py = os.path.join(sys.prefix, 'Scripts')
-    bin_user = os.path.join(user_site, 'Scripts')
     # buildout uses 'bin' on Windows too?
     if not os.path.exists(bin_py):
         bin_py = os.path.join(sys.prefix, 'bin')
         bin_user = os.path.join(user_site, 'bin')
 else:
     bin_py = os.path.join(sys.prefix, 'bin')
```

Comment:
I think this can be savely set to the same value as `bin_user`, as outside of a venv
```
>>> import os, sys, sysconfig
>>> os.path.join(sys.prefix, 'bin')
'/usr/bin'
>>> sysconfig.get_path("scripts")
'/usr/bin'
```
and in a venv
```
>>> import os, sys, sysconfig
>>> os.path.join(sys.prefix, 'bin')
'/home/user/foobar/bin'
>>> sysconfig.get_path("scripts")
'/home/user/foobar/bin'
```

Reply:
It might but there's a lot of edge cases that I don't know about. Let's wait for maintainer feedback before touching more stuff.

Reply:
It is currect that `bin_py` can also be replaced. But that is refactoring work unrelated to the issue at hand, and IMO better to be submitted as a separate PR.